### PR TITLE
Pride ties

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

Pride ties
Inhands by any% and on mobs were done by @HighPrincessErinys
image

## Why It's Good For The Game

its so gay i love it

## Changelog
:cl:
add: adds new LGBT pride ties
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
i would like to add this is now the 6th time ive gotten distracted in a row

**Comment by HighPrincessErinys** [comment]
its so gay i love it

**Comment by Sun-Soaked** [comment]
our power grows

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -43,7 +43,18 @@
...
+		/obj/item/clothing/neck/tie/trans = 3,
+		/obj/item/clothing/neck/tie/pan = 3,
+		/obj/item/clothing/neck/tie/enby = 3,
+		/obj/item/clothing/neck/tie/bi = 3,
+		/obj/item/clothing/neck/tie/lesbian = 3,
+		/obj/item/clothing/neck/tie/intersex = 3,
+		/obj/item/clothing/neck/tie/gay = 3,
+		/obj/item/clothing/neck/tie/genderfluid = 3,
+		/obj/item/clothing/neck/tie/asexual = 3,
+		/obj/item/clothing/neck/tie/genderfae = 3,
```
Comment (MarkSuckerberg) [inline code/modules/vending/clothesmate.dm]: Hhhhhhhh

Comment (rye-rice) [reply code/modules/vending/clothesmate.dm]: I mean... had to make them accessible somehow, but could move them to loadouts

Comment (MarkSuckerberg) [reply code/modules/vending/clothesmate.dm]: I would much prefer that, and remind me at some point to make loadouts have the ability to have like a dropdown selection or something

Comment (ZephyrTFA) [reply code/modules/vending/clothesmate.dm]: Instead of doing this, make it possible to reskin a tie by Alt clicking it

**Comment by ToasterBan** [comment]
the idea is cute but there's the problem that ties suck ass and scarves are superior

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
Instead of adding all the ties to the vendor, have it possible to reskin the ties via AltClick

**Comment by rye-rice** [comment]
This is now reskinnable

**Comment by ZephyrTFA** [comment]
Cant merge it until your maps compile.
Open them in SDMM and fix the pathing.
Its highly likely that you changed a path for ties somewhere that is no longer resolving in maps

**Comment by rye-rice** [comment]
the issue with map checks seems to be unrelated
fixing in #323

**Pull request merged**